src/server/exit-page/index.njk

```diff
@@ -0,0 +1,35 @@
+{% extends 'layouts/page.njk' %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-body" data-testid="app-page-body">
+        <div class="gem-c-title govuk-!-margin-top-0 govuk-!-margin-bottom-8">
```

Comment:
In the prototype, we're missing this wrapper & I can't see it in the [GDS design system](https://design-system.service.gov.uk/styles/headings/).  

Worth removing?

Reply:
hmmm woder where i got this from....will have a look

Reply:
now follow [this layout](https://design-system.service.gov.uk/styles/layout/)